tests/PySys/log_request_end_to_end/create_test_logs.py

```diff
@@ -0,0 +1,62 @@
+import os
+from datetime import datetime, timedelta
+from random import randint, shuffle
+from typing import Optional
+from retry import retry
+# this test will look at the date of current files in /var/log/tedge/agent/
+# and create example files with the same date.
+
+def read_example_log():
+    if not os.path.isdir("/var/log/tedge/agent"):
+        return None
+
+    content = ""
+    for filename in os.listdir("/var/log/tedge/agent/"):
+        if "software-list" in filename:
+            content += open(f"/var/log/tedge/agent/{filename}", "r").read()
+    return content.splitlines()
+
+
+ERROR_MESSAGES = [
+    f"Error: in line {randint(1, 10000)}.",
+    "Error: No such file or directory: /home/some/file",
+    "Error: Connection timed out. OS error 111.",
+    "Error: Is MQTT running?",
+    "Error: missing dependency mosquitto-clients."
+]
+
+
+def create_fake_logs(bad_lines_ratio=.3, num_lines=100) -> str:
```

Comment:
Nice utility. Another option would have been to have these dummy log files pre-created and kept as test inputs in the `Input` directory but rather decided to generate it on demand. Any reason why we didn't go for that option? Was there a requirement to keep the contents really dynamic from one test invocation to another.

Reply:
Could be done, thought this way it's better. This copies some messages also from the software-list operation file as well.

Reply:
@albinsuresh, line 8 (`seed 100`) now ensures the file stays there same across different test invocations